### PR TITLE
next/630/20241107/v1

### PR DESCRIPTION
**Pull request by victorjulien**
Staging:
 - #12045
 - #12093

### CODE REVIEW

File: src/suricata.c [hunk 1]
```diff
@@ -2670,6 +2670,8 @@ int PostConfLoadedSetup(SCInstance *suri)
 
     MacSetRegisterFlowStorage();
 
+    SigTableInit();
```
Comment (jasonish) [inline src/suricata.c:2673]: Oh, I bet the fuzzers require an update due to this.

Comment (victorjulien) [reply src/suricata.c:2673]: @catenacyber can you do a PR to fix this post-merge of the PR?

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 23300

File: src/suricata.c (hunk 1 above)
Comment (jasonish) [reply src/suricata.c:2673]: Trivial patch: https://github.com/OISF/suricata/pull/12109/commits/1e13a3fa5eb5a7cbbc08fe5d1b59edd056863db0#diff-f327f4d38722d8a1e01f7ded5d72692b48b5a77d450f4995019e791af4182d07

**Comment by victorjulien** [comment]
Needs additional fixes.

**Pull request closed without merging**